### PR TITLE
Fixed Bugs

### PR DESCRIPTION
**Pull request by TheEmperor342**
- ban command
- togglemod

### CODE REVIEW

**Review by itsmebirdie — COMMENTED** [review]
Acknowledged

**Review by itsmebirdie — COMMENTED** [review]
Bot is running all fine without any interruptions, but please check on this:
image

Happening when using profane word and there is no **muted** role. Fixes are appreciated.

**Comment by itsmebirdie** [comment]
On `unban` command, returns the error:

**Comment by itsmebirdie** [comment]
On `unban` command, returns this:
image

**Comment by TheEmperor342** [comment]
> On `unban` command, returns this:
> image

Intentional

You should enter the name#discriminator to unban the user

**Comment by itsmebirdie** [comment]
> > On `unban` command, returns this:
> > image
> 
> Intentional
> 
> You should enter the name#discriminator to unban the user

Well, that's very confusingly interrelated way to unban :(

**Comment by itsmebirdie** [comment]
No spacing between username and the unban term.

File: Cogs/moderation.py [hunk 1]
```diff
@@ -10,7 +11,7 @@ class Logs(commands.Cog):
     def __init__(self,bot):
         self.bot=bot
         with open("Configuration/ModConfig.json") as f: self.CONFIG = json.loads(f.read())
-        self.illegal_words=['Nigger','Nigga','N1gg3r','N1gger','Nigg3r','N1gga','N1gg@','Dick','Fuck','F U C K','f u c k','gandu','gaandu','gaamdu','fuck','nigger','nigga','n1gg3r','n1gga','n1gg@','dick']
+        self.illegal_words=['test','Nigger','Nigga','N1gg3r','N1gger','Nigg3r','N1gga','N1gg@','Dick','Fuck','F U C K','f u c k','gandu','gaandu','gaamdu','fuck','nigger','nigga','n1gg3r','n1gga','n1gg@','dick']
```
Comment (itsmebirdie) [inline Cogs/moderation.py:14]: Please remove the word "*test*" from illegal words :D

**Review by itsmebirdie — COMMENTED** [review]
Acknowledged for testing.

File: Cogs/moderation.py (hunk 1 above)
Comment (TheEmperor342) [reply Cogs/moderation.py:14]: oh yes i forgot

Comment (itsmebirdie) [reply Cogs/moderation.py]: I have changed it don't worry :D

**Comment by itsmebirdie** [comment]
Approved for merge.
Note: Assistant bot will now update only on new releases instead of new commits to ensure all the codebase is perfect and ready to ship.

**Pull request merged**